pubtools/pulplib/_impl/model/unit/rpm.py

```diff
@@ -54,6 +54,12 @@ class RpmUnit(Unit):
         :meth:`~pubtools.pulplib.Repository.signing_keys`
     """
 
+    filename = pulp_attrib(default=None, type=str, pulp_field="filename")
+    """
+    Filename of this RPM.
```

Comment:
Minor - there ought to be a blank line after this.

It's like git commit messages, the first line will be used as a summary in some places.